### ISSUE TITLE
SteamVR stuck at "Headset Display Disconnected" with Valve Index headset

### ISSUE THREAD

**Issue by kdashg** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#184]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

SteamVR

### Processor / Processor Number [Required]

AMD 7950x

### Graphic Card [Required]

Intel A770

### GPU Driver Version [Required]

- [ ] 31.0.101.3793 (*latest*)
- [ ] 31.0.101.3491
- [X] 31.0.101.3490
- [ ] 31.0.101.3430
- [ ] 31.0.101.3276
- [ ] 30.0.101.3268
- [ ] 30.0.101.3259
- [ ] 30.0.101.1743

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H1 (*October 2021 Update*)
- [ ] Windows 10 21H2 (*November 2021 Update*)
- [ ] Windows 10 21H1 (*May 2021 Update*)
- [ ] Windows 10 20H2 (*October 2020 Update*)
- [ ] Other (Please specify)

### Intel System Support Utility report

[intel-ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/9998983/intel-ssu.txt)

### Description and steps to reproduce [Required]

With:
* Steam and SteamVR installed
* Valve Index headset plugged in

Do:
* Within Steam, start SteamVR
* SteamVR starts, but says "Headset Display Disconnected"
* Hit "Reset Headset"
* SteamVR shuts down, and starts up again, but says "Headset Display Disconnected"

**Comment by IntelSupport-Rozilah** [comment]
hi @kdashg,
Thanks for contacting Intel, just for confirmation, did you try on latest [Beta version](https://www.intel.com/content/www/us/en/download/729157/intel-arc-graphics-windows-dch-driver-beta.html)?

**Comment by kdashg** [comment]
Yes, trying the Beta version 31.0.101.3793 has the same behavior.

I suspect this in Intel trying too hard to catch when the headset display is re-added to the system, and doing something with it before the steamvr driver can catch it instead, to do its special thing (where it makes it not be treated as part of the extended desktop).

It could also very well be SteamVR's fault, I can't tell.

**Comment by kdashg** [comment]
I have filed a support ticket with Steam on the SteamVR application side as well.

**Comment by IntelSupport-Rozilah** [comment]
hi @kdashg,
There is new beta release, could you try it [31.0.101.3802](https://www.intel.com/content/www/us/en/download/729157/intel-arc-graphics-windows-dch-driver-beta.html), let me know your finding.

**Comment by IntelSupport-Arun** [comment]
Hi @kdashg,  Did you get any chance to test the latest driver [31.0.101.3802](https://www.intel.com/content/www/us/en/download/729157/intel-arc-graphics-windows-dch-driver-beta.html)?

**Comment by ckcollab** [comment]
@IntelSupport-Arun @IntelSupport-Rozilah hey guys, thanks so much for providing support for this!

I can confirm with driver `31.0.101.3802` (BETA or normal, same build number so maybe they're the same thing?) that I experience this problem. For other Googler's to find this, here's a more complete paste of the error string:

> Error message is "Your graphics card cannot successfully connect to your HMD display"

I've tried multiple HDMI ports, display ports with an HDMI adapter, etc. and nothing works. We're using an old HTC Vive. The _exact_ same setup was working with a GTX1060, so I am fairly convinced it's the drivers/intel arc graphics card that is the problemo.

Appreciate any help, love more competition in this space! I am very down to hop in a video call to assist in debugging, pulling up stack traces, etc. :)

**Comment by IntelSupport-Arun** [comment]
@hi @kdashg,
Did you get any chance to test the latest driver [31.0.101.3802](https://www.intel.com/content/www/us/en/download/729157/intel-arc-graphics-windows-dch-driver-beta.html)? 
Also, any update from Steam on the SteamVR application side?

**Comment by IntelSupport-Rozilah** [comment]
hi @kdashg,

The issue was verified internally and we will be working on adding support for the Valve Index fix. I want to give you a heads-up that the fix may take 3 to 6 months to be included

**Comment by ckcollab** [comment]
> may take 3 to 6 months to be included

Oh no, bummer! Will it be on beta sooner than that? Appreciate the quick answers: it's okay if it will take time, honest timelines better than fake ones!

**Comment by IntelSupport-Rozilah** [comment]
hi all,
For the latest update from our internal, Intel Arc graphics support for VR headsets continues to evolve as we ramp our products and add updates to our software stack. At this time we are unable to provide a target date for when each VR headset will be enabled. So we will close the case at this moment first.

**Comment by IntelSupport-Rozilah** [comment]
@IGCIT , 
need your help to close the issue at this moment.

**Comment by MyChaOS87** [comment]
> hi @kdashg,
> 
> The issue was verified internally and we will be working on adding support for the Valve Index fix. I want to give you a heads-up that the fix may take 3 to 6 months to be included

@IntelSupport-Rozilah any ETA now, its 3 months later (with current beta drivers 31.0.101.4148)

I found this out today, as I specifically bought a system as VR system... My normal PC was too much to always move 2 stories into the VR room... Total Bummer... the A770 doesn't work at all. So the NUC 12 Enthusiast is basically a paperweight ... (huge disappointment as you can imagine)

Is there any way we could help in getting that fixed? Alpha drivers to test or something?
